API.md

```diff
@@ -200,8 +200,8 @@ const category = {
     id:string,
     userId:string,
 	products:[{
-		productId:string,
-		quantity:number,
+		product:Product,
```

Comment:
Product fields will be:
id
title
shortSubtitle
longSubtitle
description
price
discountPercentage

Please confirm

Reply:
add : smallImageUrl
remove: longSubtitle

Reply:
Confirmed
@haxick